### DISCUSSION TITLE
H3-Py: polyfill_polygon code and documentation?

### DISCUSSION THREAD

**Discussion by diehl** [uber/h3#803] [Q&A] [answered]
While inspecting code in the H3-Pandas library, I ran across a call to `polyfill_polygon` in the H3-Py library. Searching the H3-Py documentation and repo, I've been unable to find any reference to that function. I'm particularly interested in the code to understand what is happening under the hood. Could someone orient me? Github and Google search are failing me.

**Comment by dfellis** [comment, marked as answer]
So this is the repo for the C library that `h3-py` binds.

But to answer your question, you didn't find `polyfill_polygon` because `h3-pandas` still depends on the V3 version of `h3-py`. If you look at the last v3 release, [you can find `polyfill_polygon` in the API definition](https://github.com/uber/h3-py/blob/f8958ac788ea04e2b383c8b859f75a05c3fcb815/src/h3/api/basic_int/_public_api.py#L63) and [in the Cython code's `geo.pyx` file](https://github.com/uber/h3-py/blob/f8958ac788ea04e2b383c8b859f75a05c3fcb815/src/h3/_cy/geo.pyx#L121)

(Also, the new Github search is garbage and can't find it since it's not in the latest commit anymore. Why they don't just wrap [OpenGrok](https://oracle.github.io/opengrok/) is beyond me.)

**Comment by diehl** [comment]
@dfellis Thank you sir! This is super helpful. Very much appreciate the quick reply.
